### ISSUE TITLE
Desktop menu does not work with keyboard navigation

### ISSUE THREAD

**Issue by SleeplessByte** [fronteers/website#192]
**Describe the bug**
Related to #184.

Using my keyboard I cannot traverse the dropdown menu.

**Comment by edwinm** [comment]
Fixed
